taxcalc/tests/test_policy.py

```diff
@@ -281,35 +280,9 @@ def test_multi_year_reform():
     check_eitc_c(pol, reform, ifactor)
     check_ii_em(pol, reform, ifactor)
     check_ss_earnings_c(pol, reform, wfactor)
-    check_ctc_c(pol, reform)
     # end of test_multi_year_reform with the check_* functions below:
 
 
-def check_ctc_c(ppo, reform):
```

Comment:
Why is this test being removed?

Reply:
Because CTC_c is now indexed

Reply:
@jdebacker ,The `test_multi_year_reform` you are asking about here still does exactly what its docstring says it should do:
```
    Test multi-year reform involving 1D and 2D parameters.
```
So, these changes do not dilute the strength of the test.

Since you haven't asked any followup questions after my response to your original question, I'm assuming that this discussion is resolved.